### PR TITLE
Mac m1 run through

### PR DESCRIPTION
**Pull request by pacostas**

### CODE REVIEW

**Review by mhdawson — APPROVED** [review]
LGTM

File: cloud-native/README.md [hunk 1]
```diff
@@ -80,16 +82,20 @@ podman system connection default podman-machine-default-root
 
 #### On Mac:
 
+1. install minikube
```
Comment (mhdawson) [inline cloud-native/README.md:85]: I think I was able to just download and install the binary?  It is necessary to use brew ?

Comment (pacostas) [reply cloud-native/README.md:85]: No, this will change

Comment (mhdawson) [reply cloud-native/README.md:85]: @pacostas not quite sure I follow.  No it's not possible to just isntall the binary or No it's not necessary to use brew?

Comment (pacostas) [reply cloud-native/README.md:85]: I wanted to say that I'll change it. It is not necessary to use brew. I'll add soon the binary version installation.
So far we have instructions on how to install minikube in mac and windows. @mhdawson Should I also add instructions on how to install minkube on the linux section too?

Comment (mhdawson) [reply cloud-native/README.md]: >  Should I also add instructions on how to install minkube on the linux section too? 

for consistency that probably makes sense. I think I just followed the instructions from the links, but if we have a section for other platforms, the having it for linux as well makes sense.

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**